README.md

```
@@ -98,6 +98,9 @@ Helpful link for beginners: [Try it out](https://docs.n8n.io/try-it-out/)
 
 ## Version history
 
+- **v1.0.10**:
+  	- Updated dependencies
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)